packages/gatsby/src/schema/build-connection-fields.js

```diff
@@ -104,7 +104,30 @@ module.exports = type => {
           groupConnections.push(groupConn)
         })
 
-        return groupConnections
+        // make tags arrays case ignored
+        const tags = groupConnections.map(e=>{
```

Comment:
sorry this one: i'm still using the old name of hte API https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-remark/src/gatsby-node.js#L2